src/services/credentialEvents/index.ts

```diff
@@ -58,6 +61,35 @@ export default class CredentialEvents {
       return
     }
 
+    if (this.isCredentialError(record)) {
+      this.logger.debug('There was an erorr in credential issuance of credential %s', record.id)
+      if (!record.errorMessage) {
+        this.logger.debug('Errror message in error report is missing for credential', record.id)
+        return
+      }
+      try {
+        const startIndex = record.errorMessage.indexOf('{')
+
+        // Extract the JSON part starting from the first '{'
+        const jsonString = startIndex !== -1 ? record.errorMessage.slice(startIndex) : null
```

Comment:
I think we should make sure we log out if `jsonString` ends up as null. Maybe throw if it's null?